block_iagora.php

```diff
@@ -48,18 +48,31 @@ public function get_content() {
 
         $this->content = new stdClass();
         $this->content->footer = '';
+        $copilotendpointurl = isset($this->config->copilotendpointurl) ? $this->config->copilotendpointurl : '';
 
-        $iframeurl = isset($this->config->iframeurl) ? $this->config->iframeurl : '';
-
-        if (empty($iframeurl)) {
-            $this->content->text = get_string('noiframeurl', 'block_iagora');
+        if (empty($copilotendpointurl)) {
+            $this->content->text = get_string('nocopilotendpointurl', 'block_iagora');
         } else {
-            $this->content->text = '<iframe src="' . $iframeurl . '" width="100%" height="400px" frameborder="0"></iframe>';
+            $this->content->text = $this->generate_chat_content($copilotendpointurl);
         }
-
         return $this->content;
     }
```

Comment:
```suggestion
    }

```
Could we add an empty line between the two functions to visually separate them?